vizro-core/src/vizro/static/css/tiles.css

```
@@ -5,6 +5,16 @@
   width: 100%;
 }
 
+
+.dash-graph {
+  /* Plotly graphs require their parent element to be visible and to have both width and height explicitly defined.
+     If the parent does not have a defined height, the graph may not render or size correctly.
+     This min-height ensures the graph is displayed even when the parent lacks an explicit height.
+     TODO: Consider increasing min-height if graphs are not sufficiently visible.
+  */
+  min-height: 100px;
```

I think we might need at least 250px, because otherwise the charts can be too small.

```suggestion
  min-height: 250px;
```